src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs

```diff
@@ -284,33 +284,33 @@ public bool TryFetch(LocalUserRegistryOptionPersister persister, OptionKey2 opti
         {"FadingOptions_FadeOutUnusedImports", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Specific.FadeOutUnusedImports")},
         {"Storage_CloudCacheFeatureFlag", new FeatureFlagStorage(@"Roslyn.CloudCache3")},
         {"Storage_Database", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Database")},
-        {"FeatureOnOffOptions_AddImportsOnPaste", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Specific.AddImportsOnPaste2")},
+        {"dotnet_add_import_on_paste_options_add_imports_on_paste", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Specific.AddImportsOnPaste2")},
```

Comment:
This ideally would need to be sorted. But for review purposes, I would keep it for the time being. And sort them in a following PR